react-datetime.d.ts

```diff
@@ -50,17 +50,17 @@ declare module ReactDatetime {
      only parameter, if the date in the input is valid. If it isn't, the value
      of the input (a string) is returned.
      */
-    onChange?:(x: string) => void;
+    onChange?:(x: string|any) => void;
```

Comment:
Doesn't this actually return something according to the docs? 

> The callback receives the selected moment object as only parameter, if the date in the input is valid. If it isn't, the value of the input (a string) is returned.

Or is the documentation a bit confusing? First it mentions the parameter, and then it says "If it isn't - return string". Maybe I'm just misunderstanding something here?